aopy/data.py

```diff
@@ -16,6 +16,34 @@
 import pickle
 import yaml
 from .utils import get_pulse_edge_times, compute_pulse_duty_cycles
+import shutil
+
+def move_files(files_list, dest_dir):
+    '''
+    This is a function that moves files/folders in files_list to dest_dir. This function is part of a backup script.
+    Args:
+        files_list (list): List of strings with filenames including full filepath
+        dest_dir (str): Destination fullpath
+
+    Returns: Nothing
+
+    '''
+    for file_to_move in files_list:
+        if file_to_move:
+            source = file_to_move[0] # assuming there will only ever be one hdf file and one ecube directory per entry, which is the case in our setup..
+            dest = os.path.join(dest_dir, os.path.basename(source))
+            # print(dest)
+            if not os.path.exists(dest):
+                print(f"Copying {dest}...", end="")
+                if os.path.isfile(source):
+                    shutil.copy2(source, dest)
+                else:
+                    shutil.copytree(source,dest)
+                print("")
+            else:
+                print(f"File {source} already moved!")
```

Comment:
i think files_list should be just a list, not a list of lists.
```suggestion
    for source in files_list:
        dest = os.path.join(dest_dir, os.path.basename(source))
        # print(dest)
        if not os.path.exists(dest):
            print(f"Copying {dest}...", end="")
            if os.path.isfile(source):
                shutil.copy2(source, dest)
            else:
                shutil.copytree(source,dest)
            print("")
        else:
            print(f"File {source} already moved!")
```